restapi/admin_config.go

```diff
@@ -141,26 +142,28 @@ func getConfigResponse(params admin_api.ConfigInfoParams) (*models.Configuration
 }
 
 // setConfig sets a configuration with the defined key values
-func setConfig(client MinioAdmin, name *string, kvs []*models.ConfigurationKV, arnResourceID string) error {
-	config := buildConfig(name, kvs, arnResourceID)
-	ctx := context.Background()
+func setConfig(ctx context.Context, client MinioAdmin, configName *string, kvs []*models.ConfigurationKV) error {
+	config := buildConfig(configName, kvs)
 	if err := client.setConfigKV(ctx, *config); err != nil {
 		return err
 	}
 	return nil
 }
 
-// buildConfig builds a concatenated string including name and keyvalues
-// e.g. `region name=us-west-1`
-func buildConfig(name *string, kvs []*models.ConfigurationKV, arnResourceID string) *string {
+func setConfigWithARN(ctx context.Context, client MinioAdmin, configName *string, kvs []*models.ConfigurationKV, arn string) error {
```

Comment:
```suggestion
func setConfigWithARN(ctx context.Context, client MinioAdmin, configName *string, kvs []*models.ConfigurationKV, arnResourceID string) error {
``` since it is not an arn

Reply:
the R in ARN stands for Resource (ARN = Amazon Resource Name) so ARNResourceId would mean amazon resource name resource identifier xD hahaha

Reply:
so not resource it should be `account_id` : from docs.
```
arn:partition:service:region:account-id:resource-id
```
that is what you are defining not a whole ARN